src/command.rs

```diff
@@ -75,9 +75,13 @@ impl SubstrateCli for Cli {
             "MAINNET" | "mainnet" => Box::new(chain_spec::mainnet::ChainSpec::from_json_bytes(
                 &include_bytes!("./chain_specs/mainnet_raw.json")[..],
             )?),
-            "TESTNET" | "testnet" => Box::new(chain_spec::mainnet::ChainSpec::from_json_bytes(
+            "TESTNET" | "testnet" => Box::new(chain_spec::testnet::ChainSpec::from_json_bytes(
```

Comment:
I am surprised this hasn't caused a problem before.  Maybe it just forced the nodes to failback to the wasm runtime instead of being able to use the native runtime, since they wouldn't have been compatible.